docs/commerce/shipping_management/configure_shipment.md

```diff
@@ -0,0 +1,41 @@
+---
+description: Configure the default shipment process.
+edition: commerce
+---
+
+# Configure shipment
+
+When you work with your Commerce implementation, you can review and modify 
+the shipment configuration.
+
+!!! note "Permissions" 
+
+    When you modify the workflow configuration, make sure you properly set user [permissions](permission_use_cases.md#commerce) to the shipping management component.
+
+## Shipment workflow
+
+Shipment workflow relies on a [Symfony Workflow](http://symfony.com/doc/5.4/components/workflow.html).
+Each transition represents a separate shipment step. 
+
+The default fallback workflow is `ibexa_shipment`, which is prepended at bundle level.
+
+### Default shipment workflow configuration
+
+The default shipment workflow configuration looks as follows. `ibexa_shipment` key is used by default, you can replace it with your custom workflow identifier if needed.
```

Comment:
Like in payment, I'd avoid quoting the default workflow here, instead, later give an example under Custom workflows